src/actions/BitPayActions.js

```diff
@@ -158,12 +158,12 @@ export async function launchBitPay(
   const requiredFeeRate = invoiceInstruction.requiredFeeRate
   const spendInfo: EdgeSpendInfo = {
     selectedCurrencyCode,
-    spendTargets: [
-      {
-        nativeAmount: instructionOutput.amount.toString(),
```

Comment:
Need to remove line 155. `yarn precommit` should have caught this with lint.

Reply:
Okay thanks I will use yarn precommit